src/realtimecontroller/RealtimeController.ts

```diff
@@ -59,7 +59,35 @@ export default interface RealtimeController {
 
   /**
    * Subscribes to changes in attendee ids in order to discover attendee ids to
-   * subscribe and unsubscribe to for volume indicator updates.
+   * subscribe and unsubscribe to for volume indicator updates. The subscribed callback is called with below parameters:
+   *
+   * `attendeeId`:
+   * Unique `AttendeeId` received from `CreateAttendee` API representing an attendee.
```

Comment:
Should we say internal Chime attendee ID created by `CreateAttendee` API?

Reply:
Yup updated it. Thanks!